.github/workflows/test_k3s.yml

```diff
@@ -12,6 +9,10 @@ on:
       - "v*"
   workflow_dispatch:
 
+defaults:
+  run:
+    shell: bash
+
```

Comment:
```
          if [[ -z "${KUBECONFIG}" ]]; then
              echo "KUBECONFIG not set as an environment variable"
              EXIT=1
          fi
```

I believe this is bash script rather than sh script. Retain this workflow files default?

Reply:
The default _is_ bash?

Reply:
Oh! Omg... yes! I must have misread docs in the past or similarly. I have been under the illusion that sh was the default for a long time.

Reply:
I think I saw shell: bash being explicitly written out all over action.yaml files and wrongly deduced it was not the default in a workflow file.